File: Models/Stat.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Threading.Tasks;

namespace UrlShort.Models

{
    public class Stat
    {
        public int Id { get; set; } 
  
        public string IpAddress { get; set; } = "";
        public string UserAgent { get; set; } = "";
        public string DeviceFamily { get; set; } = "";
        public string BrowserName { get; set; } = "";
        public string BrowserVersion { get; set; } = "";
        public string OperatingSystem { get; set; } = "";
        public string OperatingSystemVirsion { get; set; } = "";

        public int ShortUrlId { get; set; }
        public virtual UrlManagment ShortUrl { get; set; }

    }
}

File: Front/src/models/Stat.ts
export interface Stat {
    Id: number; 
  
    IpAddress: string;
    UserAgent: string;
    DeviceFamily: string;
    BrowserName: string;
    BrowserVersion: string;
    OperatingSystem: string;
    OperatingSystemVirsion: string;

    ShortUrlId: number;
   // export virtual UrlManagment ShortUrl { get; set; }
    CreatedDate: Date | string;


}